### DISCUSSION TITLE
Cover URL Change?

### DISCUSSION THREAD

**Discussion by Anxiety1337** [AzuraCast/AzuraCast#4511] [Q&A]
Hi there,
I wanted to ask if it is possible that I can make the cover url custom called e.g. (cover.name.com/stationname/image.png)

**Comment by BusterNeece** [comment]
@ByGatow If you edit a station's profile, you can set a default album art URL for it. Otherwise, the system will use the default URL specified or the one uploaded for the entire installation via the Custom Branding page.
